application/app/controllers/upload_files_controller.rb

```diff
@@ -4,16 +4,16 @@ class UploadFilesController < ApplicationController
   include LoggingCommon
   include DateTimeCommon
 
-  def files
+  def index
```

Comment:
controller actions renamed to follow Rails conventions and match the `resources` route statements